Pod/Classes/SeafRepos.m

```diff
@@ -407,4 +407,21 @@ - (BOOL)editable
 {
     return NO;
 }
+
+- (void)createLibrary:(NSString *)newLibName block:(void(^)(bool success, id repoInfo))completeBlock {
+    NSString *requestUrl = [NSString stringWithFormat:API_URL"/repos/"];
+    NSString *form = [NSString stringWithFormat:@"name=%@&desc=%@",newLibName,@"new library"];
```

Comment:
space between ',' and the following variable.
Certain characters should be replace with percent-escape sequence based on the encoding specified in the newLibName. Use [newLibName escapedUrl]